tests/behat/features/campaign_plus/02-create-basic-campaign.feature

```diff
@@ -0,0 +1,34 @@
+Feature: Creation of basic campaigns
+  In order to create basic campaigns in DDB CMS
+  As an editor
+  I want to create campaigns and verify that they are triggered correctly when users
+  browse and search the site
+
+  Background:
+    Given I am logged in as a cms user with the administrators role
+
+  @api @campaign_plus @regression @cci
+  Scenario: Create a 'basic' campaign and show it on a page
+    Given "Side" content:
+      | title         | field_ding_page_body     |
+      | Campaign Page | Some content for testing |
+      | Other Page    | Some content for testing |
+    When I go to the create campaign plus page
+    And I fill a campaign with the following:
+      | Title | Behatman & Robin         |
+      | Type  | Text only              |
+      | Text  | Bats 4ever             |
+      | Link  | <front>                |
+      | Style | Boks                   |
+      | Tags  | campaign, super heroes |
+    And I add a basic trigger with the following:
+      | Rule type  | Side     |
+      | Rule value | Campaign |
+    And I save the campaign
+    When I go to "admin/content"
```

Comment:
Pathauto doesn't get set for content I created by the `Given "Side" content` step so we can't predict the path. This means I cannot use the `And I am on "[path]"` step because we don't know either `$nid` or path.

I'm not aware of a step that allows me to go to a page by title. We could make our own but not sure I see the need.